keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -193,6 +207,22 @@ not supported by that DRA driver. When that component detects problems, it can
 check its policy configuration and decide to take devices offline by creating
 a DeviceTaintRule with a taint for affected devices.
 
+#### Safe Pod Eviction
```

Comment:
How do we handle the races between scheduling and post-allocation taints? For example:

- Taint appears after scheduling but before kubelet admission.
- Taint appears while pod is running.

Reply:
IMHO `NoSchedule` does not belong into this section because the use case is about eviction.

I've extended the story with additional comments about that race.

I'm intentionally not using the `NoExecute` constant here because at this point I don't want to educate readers about the API details.